bundle.yaml

```diff
@@ -1,3 +1,74 @@
+apiVersion: apps/v1
```

Comment:
Should we add this to bundle.yaml? IIUC we can't wire it the Kube API because of missing TLS certificates. cc @paulfantom since you've requested this to be added to the bundle :)

Reply:
I completely forgot about CA provisioning when writing my initial request :facepalm: Let's not include this in `budle.yaml` unless/until we find a way to automatically create that CA.

Thanks Simon for catching this :+1: